### ISSUE TITLE
Why do not reset cached anchor/feature to None in a new clip

### ISSUE THREAD

**Issue by zhui064** [HorizonRobotics/Sparse4D#74]
In instance_bank.py, the function `reset` is only called at the first time entering the function `get`. But for a new clip, cached anchor/feature is not reset to None. Instead, cached anchor/feature of last clip is used. It maybe a bug  but almost no impact on effect ?

**Comment by linxuewu** [comment]
Yes, this is a bug, but it doesn't affect the outcome significantly. 

The correct approach is to use `instance_bank.mask` for new clips, setting the temporal anchor and feature as the current anchor and feature in the temporal cross attention.
